specification/recoveryservices/resource-manager/readme.md

```diff
@@ -74,3 +74,24 @@ csharp:
   clear-output-folder: true
 ```
 
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+
+```yaml $(python)
+python:
+  azure-arm: true
+  namespace: azure.mgmt.recoveryservices
+  package-version: 0.1.0
+  output-folder: $(output-folder)Generated/Python
+  payload-flattening-threshold: 2
+  license-header: MICROSOFT_MIT_NO_VERSION
+```
+
+## Suppression
+
+``` yaml
+directive:
+  - suppress: DefinitionsPropertiesNamesCamelCase
+    reason: Autorest invalidates two letter acronyms as well and changes in data contracts require service wide changes and require more time

```

Comment:
Yes, we have requested approval for this supprestion already and also for the XmsExamplesProvidedValidation, as said by @ravbhatnagar that "As Samer said, we cant give an exception for the examples violation. This will not block the PR to be merged but it will be shown as a violation till it is fixed." we had a mail thread with azure-spc-violations@microsoft.com, adxsr@microsoft.com with subject: Violations Suppression requested for PR 1139